i18n/languages/en.json

```diff
@@ -39,6 +39,10 @@
     "Binds.Sections.Flight.Rotate.Description": "Hold this key while using left and right to rotate",
     "Binds.Sections.Flight.SpinDirection": "Spin Direction",
     "Binds.Sections.Flight.SpinDirection.Description": "Changes which way the TARDIS rotates while flying",
+    "Binds.Sections.Flight.Tracking": "Tracking",
+    "Binds.Sections.Flight.Tracking.Description": "Hold to enable tracking selection mode",
```

Comment:
Not sure about the wording here, I wouldn't understand it if I were 
Perhaps we should have smth like "Hold to select the entity to track / follow"?

Reply:
https://github.com/MattJeanes/TARDIS/pull/969/commits/2c72beec1d2ad53220acf32989f31d755e0c3e72